packages/core/stories/card.stories.tsx

```diff
@@ -42,10 +42,8 @@ const Examples = () => (
     <ExampleRow name="Default">
       <Card>
         <div>
-          <h1 style={{ margin: "0", lineHeight: "1.3em" }}>
-            Card with Density
-          </h1>
-          <span>Here is some content</span>
+          <H1>Card with Density</H1>
```

Comment:
Let's use another Heading inside the card. I know this is just an example, but it'll never be valid in real life. Let's use an H3 styledAs H1 if you wish